.changeset/flat-rivers-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-modal": patch
```

Comment:
I think it's reasonable to assume nobody is depending on the broken focus trap behavior, so I think this is a pretty pure non-breaking bugfix